src/ArrayToTextTable.php

```diff
@@ -13,42 +13,57 @@ class ArrayToTextTable
      * @var array
      */
     private $data;
-    
+
+    /**
+     * @var bool
+     */
+    private $paddingRight = true;
+
+    /**
+     * @var array
+     */
+    private $format;
+
     /**
      * @var array
      */
     private $columnsList = [];
-    
+
     /**
      * @var int
      */
     private $maxLineLength = 40;
-    
+
     /**
      * @var array
      */
     private $columnsLength = [];
-    
+
     /**
      * @var array
      */
     private $result = [];
-    
+
     /**
      * @var string
      */
     private $charset = 'UTF-8';
-    
+
     /**
      * @var bool
      */
     private $renderHeader = true;
-    
-    public function __construct(array $data)
+
+    public function __construct(array $data, ?array $format = null)
```

Comment:
would be MUCH better to add ?array $options = [] so we could reuse it later and don't break backward compatibility. @8ctopus could you please fix it? I'll merge it then and push a new tag to github

Reply:
I've done it slightly differently because the $format array should not be empty.